numpy/core/tests/test_numeric.py

```diff
@@ -216,6 +216,9 @@ def test_var(self):
             assert_(np.isnan(np.var([])))
             assert_(w[0].category is RuntimeWarning)
 
+        B = np.array([None, 0])
```

Comment:
The test is great. As a final suggestion, could you add a comment along the lines of
```
# Regression test for gh-13177, that object arrays are treated correctly.
B = np.array([1j, 0], dtype=object)
```
(Note: might as well simplify the definition...)

Reply:
I don't see that this was resolved

Reply:
strange, I could have sworn that I saw the comment there when I approved...  Anyway, not too big a deal.